client/src/components/BackToTop/BackToTop.stories.js

```diff
@@ -4,6 +4,11 @@ import React, { Fragment } from "react";
 
 import { BackToTop } from "./BackToTop.js";
 
+function focusBTT () {
+  document.body.scrollTop = document.documentElement.scrollTop = 0;
```

Comment:
I think we can keep this in `handleClick()` since we don't really have a use case where we'd want it to scroll only partially up just yet.

Also to clarify, focus is purely for focusing on a certain item within the page after the `BackToTop` button is pressed, so it wouldn't make sense to leave it here anyways haha